pyiron/vasp/base.py

```diff
@@ -942,8 +942,29 @@ def calc_minimize(
             self.write_charge_density = retain_charge_density
         if retain_electrostatic_potential:
             self.write_electrostatic_potential = retain_electrostatic_potential
-        self.set_convergence_precision(ionic_energy_tolerance=ionic_energy_tolerance,
-                                       ionic_force_tolerance=ionic_force_tolerance)
+        if ionic_forces is not None:
```

Comment:
Is there a specific reason why we do not include this part in `set_convergence_precision()` - I guess it is the same issue for all DFT codes.

Reply:
Because I realize that we overwrite the already set electronic convergence values if we call this.

Reply:
I agree with @jan-janssen - I guess it's better to unify it in `set_convergence_precision` if it's anyway doing the same thing here. Why not replacing

```
self.input.incar["EDIFF"] = electronic_energy
```

by

```
if electronic_energy is not None:
    self.input.incar["EDIFF"] = electronic_energy
```

and call:

```
self.set_convergence_precision(ionic_energy_tolerance=ionic_energy_tolerance,
                                                      ionic_force_tolerance=ionic_force_tolerance,
                                                      electronic_energy=None)
```

Reply:
Thanks, this is definitely the more elegant solution.